### ISSUE TITLE
Second click at reply or quote anchor breaks the channel (multiple `?msg=` in URL)

### ISSUE THREAD

**Issue by myfonj** [RocketChat/Rocket.Chat#6050]
Rocket.Chat Version: 0.52.0
Node Version: v4.7.3

### Steps for reproduce:
- (Enter channel, write a message)
- Reply or quote the previous message [1] and post it
- Click the time link at the end of the last message
- Click it or any other similar link in the channel again

### Expected outcome
- referenced message blinks

### Actual outcome
- blank content below channel title and infinite loading animation

### Analysis
- First click appends `?msg=xxx` to the channel URL, second also so final url is `../channel/name?msg=xxx?msg=yyy` what screws the routing

### Fix?
Omitting second and third parameter at [main.js#L77](https://github.com/RocketChat/Rocket.Chat/blob/55b14e3924c6e7c1e065295c49f1a552f177ae2e/packages/rocketchat-ui-master/client/main.js#L77) :
```
-- FlowRouter.go(link.pathname + link.search, null, FlowRouter.current().queryParams)
++ FlowRouter.go(link.pathname + link.search)
```

seemingly fixes this issue (tested locally by proxy-faking amended JS), but I'm not 100% sure it will not break something else elsewhere.

---

[1] btw, I can see absolutely no difference between reply and quote ATM

**Comment by localguru** [comment]
I can confirm that on RC 0.52.0. Reloading/refreshing the browser shows the channel content again.

**Comment by myfonj** [comment]
Seems fixed in RC 0.59.1 (or earlier).
